apps/backend/src/mcp/tools/device_management.py

```
@@ -26,6 +26,150 @@
+            device_data["location"] = location
+        if ip_address:
+            device_data["ip_address"] = ip_address
+        if ssh_port:
+            device_data["ssh_port"] = ssh_port
+        if ssh_username:
+            device_data["ssh_username"] = ssh_username
+        if tags:
+            device_data["tags"] = tags
+        
+        # Make HTTP request to the API endpoint
+        api_url = f"http://localhost:9101/api/devices"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)